git/git.go

```diff
@@ -367,12 +367,12 @@ func RunClone(cloneURL string, args []string) (target string, err error) {
 	return
 }
 
-func AddUpstreamRemote(upstreamURL, cloneDir string, branches []string) error {
-	args := []string{"-C", cloneDir, "remote", "add"}
+func AddNamedRemote(url, name, dir string, branches []string) error {
```

Comment:
This now has large overlap with the existing `git.AddRemote()` function, but it's fine for now as we can clean that up as part of https://github.com/cli/cli/pull/4900